internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -111,7 +111,13 @@ Resources:
     {{- end}}
     {{- end}}
     {{- if .NLB}}
+    {{- if eq (len .NLB.Listener) 1 }}
       - NLBListener
+    {{- else if gt (len .NLB.Listener) 1 }}
+    {{- range $listener := .NLB.Listener }}
+      - NLBListenerFor{{$listener.Port}}
+    {{- end }}
+    {{- end }}
```

Comment:
nit: I think we can perhaps simplify this slightly we adopt a different naming pattern:
```
{{- range $i, $listener := .NLB.Listener }}
- NLBListener{{- if ne $i 0}}{{$i + 1}}{{- end}}
{{- end }}
```

Reply:
I like this naming. Just want to make sure if there is any intent behind adding `1` to every `$i` in the loop? Because we are anyway keeping 1st resource as `NLBListener` or `NLBTargetGroup` and then appending index to it which would start from 1. In case when we will have 2 listeners then we will get `NLBListener` and `NLBListener1` which seems fine to me. OR you want `NLBListener` and `NLBListener2` for some reason?

Reply:
oh I like your suggestion even more! starting from 2 :shipit: but yea that was why I was doing +1